### ISSUE TITLE
Increasing Loss and Validation Loss

### ISSUE THREAD

**Issue by hnahi** [breizhn/DTLN#58]
Hello Everyone.

I was trying to train the code with all the default set-ups but while training, both of the loss and validation loss are increasing. Anyone else faced this problem and solved it anyhow, please let me know.

Thanks in advance.

**Comment by jeungmin717** [comment]
@hnahi 
have you got any improvements?
I am also having the same issues 
my val loss is higher than 49.
